spec/disemvowling.js

```diff
@@ -0,0 +1,13 @@
+// import chai, { expect } from 'chai'
```

Comment:
I recommend not submitting commented out files in your PR - it just introduces noise into the commit.  Consider stashing a file, instead of commenting it out, prior to issuing your PR.